### DISCUSSION TITLE
Change of default fonts

### DISCUSSION THREAD

**Discussion by LC-Zorg** [OpenTTD/OpenTTD#10279] [General]
This discussion mainly refers to #10278 (allow to change the font by the console command). I wanted to post this only as a comment in that already closed PR, but I think it's a bit more important because it relates to more than one reported issue.

Guys, sorry, I appreciate your work, but this change hardly solves anything. Note that the vast majority of players have no idea about any commands. This means that for those who need such a function the most, nothing will change. Don't get me wrong, this isn't meanness on my part, but in this way, at the moment, you have introduced a solution only for yourself and a very small group of more advanced players.

This doesn't close #8596 (allow changing of fonts ingame)
This doesn't close #9277 (duplicate report by title, although it is actually about unreadable small font)
This doesn't close #10091 (unreadable, tiring mono font in readme, changelog and license files)
This doesn't close #10153 (unreadable small font in many interface elements)

Please, consider changing the default fonts to ones that would be scalable with the rest of the interface and readable at any screen resolution, because this is the source of all the problems reported above, including the first one. After the implementation of the new way of resizing the GUI, the ability to change the font in the game will not be needed if the default settings will be good. I would like to point out here that OTTD is the only game where I felt the need to change the font. In every other game I know, the font size is always proportional to the rest of the GUI no matter if I play on small or huge screen.

Cheers and Merry Christmas! :)

**Comment by LordAro** [comment]
No one suggested it was a complete solution. A graphical way of specifying fonts (specifically getting a list of fonts from the system) is much, much harder to do in a way that works cross-platform.

And yes, this should have been a comment on the PR or even original issue.

**Reply by LordAro** [reply]
But I tell you what, I'll reopen #8596, reword it slightly, and close #9277

**Reply by LordAro** [reply]
Just because it's Christmas
